rest-api/api_util.py

```diff
@@ -122,7 +122,7 @@ def get_site_id_from_google_group(obj, site_dao):
   return None
 
 def get_awardee_id_from_name(obj, hpo_dao):
-  if 'awardee' in obj:
+  if 'awardee' in obj and obj['awardee'] != 'UNSET':
```

Comment:
So: there is an UNSET hpo row in the database, and hpoId on participant is not nullable... we set hpoId to UNSET (0) if a participant is unpaired.

However, organizationId and siteId are nullable and there is no UNSET organization or site.

So I would have maybe expected get_site_id_from_google_group and get_organization_id_from_external_id to do these UNSET checks... but not necessarily this function.

Also, here and below, use the UNSET constant:

https://github.com/all-of-us/raw-data-repository/blob/master/rdr_common/code_constants.py#L4

rather than putting 'UNSET' in the code.

Reply:
done

Reply:
I don't think you addressed the first part of this comment... should we be checking for UNSET in get_site_id_from_google_group and get_organization_id_from_external_id too?